frontend/templates/views/partials/author.j2

```diff
@@ -44,11 +32,6 @@
            , 
```

Comment:
Please remove the space before the comma

Reply:
Done.

I put a space _after_ the comma on purpose. (The space before was a mistake.) I kept hoping that putting no space before the comma would mean no space in the template. Sadly, this wasn't true... my indentation creates a space.

I suppose this might be why the original code for multiple contributors was mushed onto a single line. I couldn't read it, so I transformed it into the multiline version you see here, then changed the logic so that it created output I preferred. Unfortunately, now there's a space before each comma. But I don't think that looks so bad. Personally I think the readable code is a worthy tradeoff.